### PR TITLE
update link handling per latest WNM updates

### PR DESCRIPTION
**Pull request by tomkralidis**
In support of https://github.com/wmo-im/wis2-notification-message/pull/56

### CODE REVIEW

**Comment by maaikelimper** [comment]
When the link-type="http://def.wmo.int/def/rel/wnm/-/deletion" will the link have an "href" to download from ? 
Otherwise this would fail:
```
data = http_session.get(link['href'],
                                    verify=verify_certs).content
```

**Comment by tomkralidis** [comment]
Yes, `href` is required by the WNM specification.

File: pywis_pubsub/subscribe.py [hunk 1]
```diff
@@ -133,7 +134,11 @@ def on_message_handler(client, userdata, msg):
 
         storage_class = STORAGES[userdata.get('storage').get('type')]
         storage_object = storage_class(userdata['storage'])
-        storage_object.save(data, filename)
+
+        if link.get('rel') == 'http://def.wmo.int/def/rel/wnm/-/deletion':
```
Comment (maaikelimper) [inline pywis_pubsub/subscribe.py:138]: "http://def.wmo.int/def/rel/wnm/-/deletion" should be "http://def.wmo.int/def/rel/wnm/-/deleted"

**Review by maaikelimper — APPROVED** [review]

File: pywis_pubsub/subscribe.py (hunk 1 above)
Comment (tomkralidis) [reply pywis_pubsub/subscribe.py:138]: Typo.  Correct value is `http://def.wmo.int/def/rel/wnm/-/deletion`.  Fixed in [spec](https://github.com/wmo-im/wis2-notification-message/pull/58) and this PR.  Good catch!

**Pull request merged**